docs/6-Handling_deviations.md

```diff
@@ -10,13 +10,11 @@ that has been exported from our namespace). Consider the case where you would us
 following config file for the `MyExampleConfig` defined before:
 
 ```toml
-name = true
 [section1]
 field1 = 4
 field2 = "22"
 ```
 
-The `bool` specified for `name` will be coerced into a `str` value of `"true"`.
 The `int` specified for `field1` will be coerced into a `float` value of `4.0`.
 The `str` specified for `field2` will be coerced into an `int` value of `22`.
```

Comment:
Regarding the previous comment, somewhere here we could point to the table showing which coercions actually happen?

Reply:
I chose to do it earlier...